Building DAG of jobs...
Using shell: /usr/bin/bash
Provided cores: 4
Rules claiming more threads will be scaled down.
Provided resources: mem_mb=1000, mem_mib=954, disk_mb=1000, disk_mib=954, mem_gb=16, time_min=119
Select jobs to execute...

[Fri Jul 19 13:16:22 2024]
rule fast_condor:
    input: condor_inputs/BPA-4/character_bin_matrix.csv, condor_inputs/BPA-4/alt_readcounts.csv, condor_inputs/BPA-4/total_readcounts.csv, condor_inputs/BPA-4/germline_mutations.txt, condor_inputs/BPA-4/somatic_mutations.txt, /data/iacobuzc/haochen/Tapestri_main_manuscript_analysis/data_compiled/falcon_solutions/BPA-4-RSX_homdel_nclones=9.unique_cn_clone_profiles.csv
    output: condor_outputs/BPA-4/out_tree.newick, condor_outputs/pickle_files/BPA-4_self.solT_cell
    log: condor_outputs/BPA-4/condor_outputs.log, condor_outputs/BPA-4/condor_outputs.err.log
    jobid: 0
    reason: Missing output files: condor_outputs/BPA-4/out_tree.newick, condor_outputs/pickle_files/BPA-4_self.solT_cell
    wildcards: patient=BPA-4
    threads: 4
    resources: mem_mb=1000, mem_mib=954, disk_mb=1000, disk_mib=954, tmpdir=/scratch/lsftmp/8217038.tmpdir, mem_gb=16, time_min=119

Activating conda environment: condor
/usr/bin/bash: line 1: 23275 Killed                  python /data/iacobuzc/haochen/Tapestri_batch2/analysis/full-ConDoR/fast-ConDoR/src/fast-condor.py -i condor_inputs/BPA-4/character_bin_matrix.csv -r condor_inputs/BPA-4/total_readcounts.csv -v condor_inputs/BPA-4/alt_readcounts.csv -s condor_inputs/BPA-4/germline_mutations.txt -s2 condor_inputs/BPA-4/somatic_mutations.txt -o condor_outputs/BPA-4/out -m /data/iacobuzc/haochen/Tapestri_batch2/analysis/full-ConDoR/references/panel3559_analysis.gene_cytoband.formatted.manual_adjusted.gene_pos_ado_added.csv -d BPA-4 --scr --cnp /data/iacobuzc/haochen/Tapestri_main_manuscript_analysis/data_compiled/falcon_solutions/BPA-4-RSX_homdel_nclones=9.unique_cn_clone_profiles.csv --subclonal_mutations /data/iacobuzc/haochen/Tapestri_main_manuscript_analysis/data_compiled/manual_subclonal_snvs/BPA-4.subclonal_mutations.yaml > condor_outputs/BPA-4/condor_outputs.log 2> condor_outputs/BPA-4/condor_outputs.err.log
[Fri Jul 19 13:20:03 2024]
Error in rule fast_condor:
    jobid: 0
    input: condor_inputs/BPA-4/character_bin_matrix.csv, condor_inputs/BPA-4/alt_readcounts.csv, condor_inputs/BPA-4/total_readcounts.csv, condor_inputs/BPA-4/germline_mutations.txt, condor_inputs/BPA-4/somatic_mutations.txt, /data/iacobuzc/haochen/Tapestri_main_manuscript_analysis/data_compiled/falcon_solutions/BPA-4-RSX_homdel_nclones=9.unique_cn_clone_profiles.csv
    output: condor_outputs/BPA-4/out_tree.newick, condor_outputs/pickle_files/BPA-4_self.solT_cell
    log: condor_outputs/BPA-4/condor_outputs.log, condor_outputs/BPA-4/condor_outputs.err.log (check log file(s) for error details)
    conda-env: condor
    shell:
        
		python /data/iacobuzc/haochen/Tapestri_batch2/analysis/full-ConDoR/fast-ConDoR/src/fast-condor.py 			-i condor_inputs/BPA-4/character_bin_matrix.csv 			-r condor_inputs/BPA-4/total_readcounts.csv 			-v condor_inputs/BPA-4/alt_readcounts.csv 			-s condor_inputs/BPA-4/germline_mutations.txt 			-s2 condor_inputs/BPA-4/somatic_mutations.txt 			-o condor_outputs/BPA-4/out 			-m /data/iacobuzc/haochen/Tapestri_batch2/analysis/full-ConDoR/references/panel3559_analysis.gene_cytoband.formatted.manual_adjusted.gene_pos_ado_added.csv 			-d BPA-4 			--scr 			--cnp /data/iacobuzc/haochen/Tapestri_main_manuscript_analysis/data_compiled/falcon_solutions/BPA-4-RSX_homdel_nclones=9.unique_cn_clone_profiles.csv 			--subclonal_mutations /data/iacobuzc/haochen/Tapestri_main_manuscript_analysis/data_compiled/manual_subclonal_snvs/BPA-4.subclonal_mutations.yaml 			1> condor_outputs/BPA-4/condor_outputs.log 2> condor_outputs/BPA-4/condor_outputs.err.log
		
        (one of the commands exited with non-zero exit code; note that snakemake uses bash strict mode!)

Trying to restart job 0.
Select jobs to execute...

[Fri Jul 19 13:20:03 2024]
rule fast_condor:
    input: condor_inputs/BPA-4/character_bin_matrix.csv, condor_inputs/BPA-4/alt_readcounts.csv, condor_inputs/BPA-4/total_readcounts.csv, condor_inputs/BPA-4/germline_mutations.txt, condor_inputs/BPA-4/somatic_mutations.txt, /data/iacobuzc/haochen/Tapestri_main_manuscript_analysis/data_compiled/falcon_solutions/BPA-4-RSX_homdel_nclones=9.unique_cn_clone_profiles.csv
    output: condor_outputs/BPA-4/out_tree.newick, condor_outputs/pickle_files/BPA-4_self.solT_cell
    log: condor_outputs/BPA-4/condor_outputs.log, condor_outputs/BPA-4/condor_outputs.err.log
    jobid: 0
    reason: Missing output files: condor_outputs/BPA-4/out_tree.newick, condor_outputs/pickle_files/BPA-4_self.solT_cell
    wildcards: patient=BPA-4
    threads: 4
    resources: mem_mb=1000, mem_mib=954, disk_mb=1000, disk_mib=954, tmpdir=/scratch/lsftmp/8217038.tmpdir, mem_gb=16, time_min=119

Activating conda environment: condor
Traceback (most recent call last):
  File "/lila/data/iacobuzc/haochen/mambaforge/bin/conda", line 13, in <module>
Terminating processes on user request, this might take some time.
    sys.exit(main())
  File "/lila/data/iacobuzc/haochen/mambaforge/lib/python3.10/site-packages/conda/cli/main.py", line 118, in main
    from ..exceptions import conda_exception_handler
  File "/lila/data/iacobuzc/haochen/mambaforge/lib/python3.10/site-packages/conda/exceptions.py", line 24, in <module>
[Fri Jul 19 13:20:04 2024]
Error in rule fast_condor:
    jobid: 0
    input: condor_inputs/BPA-4/character_bin_matrix.csv, condor_inputs/BPA-4/alt_readcounts.csv, condor_inputs/BPA-4/total_readcounts.csv, condor_inputs/BPA-4/germline_mutations.txt, condor_inputs/BPA-4/somatic_mutations.txt, /data/iacobuzc/haochen/Tapestri_main_manuscript_analysis/data_compiled/falcon_solutions/BPA-4-RSX_homdel_nclones=9.unique_cn_clone_profiles.csv
    output: condor_outputs/BPA-4/out_tree.newick, condor_outputs/pickle_files/BPA-4_self.solT_cell
    log: condor_outputs/BPA-4/condor_outputs.log, condor_outputs/BPA-4/condor_outputs.err.log (check log file(s) for error details)
    conda-env: condor
    shell:
        
		python /data/iacobuzc/haochen/Tapestri_batch2/analysis/full-ConDoR/fast-ConDoR/src/fast-condor.py 			-i condor_inputs/BPA-4/character_bin_matrix.csv 			-r condor_inputs/BPA-4/total_readcounts.csv 			-v condor_inputs/BPA-4/alt_readcounts.csv 			-s condor_inputs/BPA-4/germline_mutations.txt 			-s2 condor_inputs/BPA-4/somatic_mutations.txt 			-o condor_outputs/BPA-4/out 			-m /data/iacobuzc/haochen/Tapestri_batch2/analysis/full-ConDoR/references/panel3559_analysis.gene_cytoband.formatted.manual_adjusted.gene_pos_ado_added.csv 			-d BPA-4 			--scr 			--cnp /data/iacobuzc/haochen/Tapestri_main_manuscript_analysis/data_compiled/falcon_solutions/BPA-4-RSX_homdel_nclones=9.unique_cn_clone_profiles.csv 			--subclonal_mutations /data/iacobuzc/haochen/Tapestri_main_manuscript_analysis/data_compiled/manual_subclonal_snvs/BPA-4.subclonal_mutations.yaml 			1> condor_outputs/BPA-4/condor_outputs.log 2> condor_outputs/BPA-4/condor_outputs.err.log
		
        (one of the commands exited with non-zero exit code; note that snakemake uses bash strict mode!)

    from .models.channel import Channel
  File "/lila/data/iacobuzc/haochen/mambaforge/lib/python3.10/site-packages/conda/models/channel.py", line 17, in <module>
    from ..base.context import context, Context
  File "/lila/data/iacobuzc/haochen/mambaforge/lib/python3.10/site-packages/conda/base/context.py", line 55, in <module>
    from ..common.configuration import (Configuration, ConfigurationLoadError, MapParameter,
  File "/lila/data/iacobuzc/haochen/mambaforge/lib/python3.10/site-packages/conda/common/configuration.py", line 41, in <module>
    from .path import expand
  File "/lila/data/iacobuzc/haochen/mambaforge/lib/python3.10/site-packages/conda/common/path.py", line 12, in <module>
    from urllib.parse import urlsplit
  File "<frozen importlib._bootstrap>", line 1024, in _find_and_load
  File "<frozen importlib._bootstrap>", line 170, in __enter__
  File "<frozen importlib._bootstrap>", line 209, in _get_module_lock
KeyboardInterrupt
